src/constants.js

```diff
@@ -177,3 +177,5 @@ export const RELEASE_SERVER = 'https://api.bitsrc.io/release';
 export const SKIP_UPDATE_FLAG = '--skip-update';
 
 export const LICENSE_FILENAME = 'LICENSE';
+
+export const ISOLATED_ENV_ROOT = path.join(os.tmpdir(), 'bit');
```

Comment:
It seems to be more BIT_TMP_DIR than an isolated environment root, maybe we can add another directory (environment) and not catch the root of `/tmp/bit` ?